sources/funtoo-http.go

```diff
@@ -32,7 +32,13 @@ func (s *funtoo) Run() error {
 		topLevelArch = "arm-64bit"
 	}
 
-	baseURL := fmt.Sprintf("%s/%s-release-std/%s/%s",
+	// Keep release backward compatible to old implementation
+	// and to permit to have yet the funtoo/1.4 alias.
+	if s.definition.Image.Release == "1.4" {
+		s.definition.Image.Release = "1.4-release-std"
+	}
```

Comment:
Do we expect 1.5 to also be `1.5-release-std`? If so, should we make the logic append `-release-std` to anything that's not `next` instead?

Reply:
At the moment there aren't plans for 1.5-release-std or other releases, just 1.4-release-std and next. Instead could be could to use it also for harvester releases that are high development trees often broken but that could be integrated eventually with this patch for cd/ci tasks.
So, I don't think that hardcoded a postfix is a great solution, IMHO.
Personally, I think that was wrong before to have statically the release-std string fixed.

Reply:
Just to complete the previous comment... the harvester releases are there:
https://harvester.funtoo.org/
and in this case, it's needed to replace the URL too but I haven't tested yet if it could work.